### PR TITLE
Initial add of the EmbeddedForm tool, which is an example of a simple…

### PR DESCRIPTION
**Pull request by stephanie-graham**
… webserver that hosts an HTML embedded form.io form rendered using USWDS.

### CODE REVIEW

File: EmbeddedForm/public/index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        <!-- Used on https://apps.form.io/uswds/ is 4.10.4 -->
+        <link rel="stylesheet" href="https://unpkg.com/formiojs@4.12.0/dist/formio.full.min.css" />
+        <script src="https://unpkg.com/formiojs@4.12.0/dist/formio.full.min.js"></script>
+        <!-- <link rel="stylesheet" href="https://unpkg.com/formiojs@4.10.4/dist/formio.full.min.css" />
+        <script src="https://unpkg.com/formiojs@4.10.4/dist/formio.full.min.js"></script> -->
+
+        <!-- Latest is 2.9.0 -->
+        <!-- Used on https://apps.form.io/uswds/ is 2.6.0 -->
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/uswds/2.9.0/css/uswds.min.css" />
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/uswds/2.9.0/js/uswds.min.js"></script>
```
Comment (Wube-T-Kifle) [inline EmbeddedForm/public/index.html:19]: This looks redundant. You already added uswds.js and uswds.min.css below.

File: EmbeddedForm/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+RUN npm install
+# If you are building your code for production
+# RUN npm ci --only=production
+
+# Bundle app source
+COPY . .
+
+EXPOSE 3000
+
+CMD [ "node", "server.js" ]
```
Comment (Wube-T-Kifle) [inline EmbeddedForm/Dockerfile]: Why do we need Node server for?

File: EmbeddedForm/public/index.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        <link rel="stylesheet" href="https://unpkg.com/@formio/uswds@1.8.2-rc.12/dist/uswds.css" />
+        <script src="https://unpkg.com/@formio/uswds@1.8.2-rc.12/dist/uswds.js"></script>
+
+        <style type="text/css">
+            body {
+                background-color: #FFCC00;
+            }
+            .pageContainer { 
+                position: absolute;
+                /* height: 80%;  */
```
Comment (Wube-T-Kifle) [inline EmbeddedForm/public/index.html]: remove dead code

File: EmbeddedForm/public/submitted.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        <title>Javascript USWDS Embedded Submitted page</title>
+
+        <style type="text/css">
+            body {
+                background-color: #FFCC00;
+                font-family: verdana;
+            }
+            .pageContainer { 
+                position: absolute;
+                /* height: 80%;  */
```
Comment (Wube-T-Kifle) [inline EmbeddedForm/public/submitted.html]: Remove this commented code.

File: EmbeddedForm/public/index.html (hunk 1 above)
Comment (stephanie-graham) [reply EmbeddedForm/public/index.html:19]: There are two different USWDS items that need to be added.  The US Government supplied uswds (the one with version 2.9.0), and the form.io one - the one with version 1.8.2-rc.12.  I was definitely confused by that too.

File: EmbeddedForm/public/index.html (hunk 3 above)
Comment (stephanie-graham) [reply EmbeddedForm/public/index.html]: done.

File: EmbeddedForm/public/submitted.html (hunk 4 above)
Comment (stephanie-graham) [reply EmbeddedForm/public/submitted.html]: done

File: EmbeddedForm/Dockerfile (hunk 2 above)
Comment (stephanie-graham) [reply EmbeddedForm/Dockerfile]: removed.  Doesn't seem to be needed.

**Comment by Wube-T-Kifle** [comment]
LGTM

**Pull request merged**